src/Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -1,20 +1,2 @@
 ; Unshipped analyzer release
-; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
-
-### New Rules
-
-Rule ID | Category | Severity | Notes
---------|----------|----------|-------
-DURABLE0001 | Orchestration | Warning | DateTimeOrchestrationAnalyzer
-DURABLE0002 | Orchestration | Warning | GuidOrchestrationAnalyzer
-DURABLE0003 | Orchestration | Warning | DelayOrchestrationAnalyzer
-DURABLE0004 | Orchestration | Warning | ThreadTaskOrchestrationAnalyzer
-DURABLE0005 | Orchestration | Warning | IOOrchestrationAnalyzer
-DURABLE0006 | Orchestration | Warning | EnvironmentOrchestrationAnalyzer
-DURABLE0007 | Orchestration | Warning | CancellationTokenOrchestrationAnalyzer
-DURABLE0008 | Orchestration | Warning | OtherBindingsOrchestrationAnalyzer
-DURABLE1001 | Attribute Binding | Error | OrchestrationTriggerBindingAnalyzer
-DURABLE1002 | Attribute Binding | Error | DurableClientBindingAnalyzer
-DURABLE1003 | Attribute Binding | Error | EntityTriggerBindingAnalyzer
-DURABLE2001 | Activity | Warning | MatchingInputOutputTypeActivityAnalyzer
-DURABLE2002 | Activity | Warning | MatchingInputOutputTypeActivityAnalyzer
\ No newline at end of file
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
```

Comment:
I am not sure what this md is used for and why we keep this md file. Can't we just remove it and only keep the shipped md?

Reply:
For analyzers, it's standard to have 2 .md files for shipped and unshipped rules. Reference: https://github.com/dotnet/roslyn-analyzers/tree/7633ceddc23f29ac9a6c844c6e5fad5f52e1ebc9/src/NetAnalyzers/Core